sdk-lib/src/main/java/cash/z/ecc/android/sdk/model/LightwalletdServer.kt

```diff
@@ -0,0 +1,5 @@
+package cash.z.ecc.android.sdk.model
+
+data class LightwalletdServer(val host: String, val port: Int, val isSecure: Boolean) {
```

Comment:
I also attach information on time limits on the channel it has been helpful to deal with server responses that were longer than they usually were due to network load. But I don't now how that works on Android's version of GRPC

Reply:
Is there a benefit of having a timeout for each endpoint?

If we make the timeouts a public API, I think that should be done separately from the endpoint.  If we support a list of endpoints in the future, the same timeouts should probably apply to all of them.

Reply:
not really. If we end up with a list of them the place for those parameters is that API call groping them all.